docs/sources/pipenv.md

```diff
@@ -0,0 +1,5 @@
+# Pipenv
+
+The pipenv source uses `pipenv` CLI command to enumerate dependencies and properties.
+
+Be sure to run `pipenv update` (or `pipenv sync`) before running `licensed` so all required packages are properly installed.
```

Comment:
just to double check, there aren't any edge cases that users need to be aware of when using this source?  It all looks pretty straightforward but worth asking 😄

Reply:
Yes, it's that simple. Where the simplicity comes from is what `pipenv` is. For `pip` one has to create the virtual environment & install the packages manually, then the `pip` source goes over what's in `requirements.txt` file. For `pipenv` it manages both the Python virtual environment & packages, hence having it setup everything (using `update` or `sync`) will yield a ready for use virtual environment with installed packages. After that we're just using `pipenv` to invoke the right `pip` to get list of all installed packages and get everything from it.